### ISSUE TITLE
Charge moves are not properly stripped

### ISSUE THREAD

**Issue by SuperFromND** [SuperFromND/iguana#5]
As pointed out by Foobs, Iguana does not currently properly strip charge commands, specifically the kind that specify a time (such as `~5a`; `~a` is fine).

To quote Elecbyte's documentation of KFM:
```ini
;   tilde (~) - to detect key releases
;          egs. command = ~a       ;release the a button
;               command = ~D, F, a ;release down, press fwd, then a
;          If you want to detect "charge moves", you can specify
;          the time the key must be held down for (in game-ticks)
;          egs. command = ~30a     ;hold a for at least 30 ticks, then release
```

We could probably just detect these and replace them with more 'normal' inputs during tokenization, shouldn't be too hard (probably).

**Comment by SuperFromND** [comment]
This has now been fixed by d798ba1 using a simple regex to strip this data. 
Maybe in the future we could make them distinguished in the movelist somehow, but this'll do for now.

**Comment by SuperFromND** [comment]
Thinking about this again now that I understand charge characters a bit better, yeah this could definitely be refined to wrap the charge input in `[]` when outputting them. There's also a proposal for a dedicated "CHARGE" glyph in Ikemen GO by default, so I might just wait and see if that gets merged.
